TTS/vocoder/utils/generic_utils.py

```diff
@@ -42,6 +42,29 @@ def to_camel(text):
     return re.sub(r'(?!^)_([a-zA-Z])', lambda m: m.group(1).upper(), text)
 
 
+def setup_wavernn(c):
+    print(" > Model: WaveRNN")
+    MyModel = importlib.import_module("TTS.vocoder.models.wavernn")
+    MyModel = getattr(MyModel, "WaveRNN")
+    model = MyModel(
+        rnn_dims=512,
```

Comment:
I think we should define model size values in config file anymore.

Reply:
We should include more model params? The "anymore" confuses me ^^. 

But yes I would say we could add more params of the model into the config to make it more adaptable.

Reply:
I've added the model params to the config file.